src/Misc/layoutbin/en-US/strings.json

```diff
@@ -124,6 +123,7 @@
     "                                   agent and should not share between multiple agents.",
     " --acceptTeeEula                   macOS and Linux only. Accept the TEE end user license agreement.",
     " --gitUseSChannel                  Windows only. Tell Git to use Windows' native cert store.",
+    " --AlwaysExtractTask               Perform an unzip for tasks for each pipeline step.",
```

Comment:
All the other options in this help use a lowercase first letter.

Reply:
Make sense, thanks 👍 Aligned it with other options